src/Doctrine/DoctrineEntityDetails.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Doctrine;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ */
+final class DoctrineEntityDetails
```

Comment:
I think this class can also be internal - i don't think you're passing it directly into the templates (which is good)